tests/unit/Statement/StatementListChangerTest.php

```diff
@@ -0,0 +1,282 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Tests\Statement;
+
+use PHPUnit_Framework_TestCase;
+use Wikibase\DataModel\Entity\PropertyId;
+use Wikibase\DataModel\Services\Statement\StatementListChanger;
+use Wikibase\DataModel\Snak\PropertyNoValueSnak;
+use Wikibase\DataModel\Statement\Statement;
+use Wikibase\DataModel\Statement\StatementList;
+
+/**
+ * @covers Wikibase\DataModel\Services\Statement\StatementListChanger
+ *
+ * @license GPL-2.0+
+ * @author Thiemo Mättig
+ */
+class StatementListChangerTest extends PHPUnit_Framework_TestCase {
+
+	public function testConstructor() {
+		$instance = new StatementListChanger( new StatementList() );
+
+		$this->assertInstanceOf( StatementListChanger::class, $instance );
```

Comment:
TDD. This is one of the first tests I wrote. It became obsolete later because all other tests are covering the same feature. I just forgot to remove it.